### ISSUE TITLE
usdview: files are not framed correctly and auto clipping is not helping much

### ISSUE THREAD

**Issue by hybridherbst** [PixarAnimationStudios/USD#2557]
### Description of Issue

Related to https://github.com/PixarAnimationStudios/OpenUSD/issues/2556, files opened in usdview aren't framed correctly independent of whether they have extents authored or not.

### Steps to Reproduce
1. Download the files from  https://github.com/PixarAnimationStudios/OpenUSD/issues/2556 and open them with usdview.
2. Two observations:
  3. they are displayed very small (not framed)
  4. when zooming, they completely disappear (incorrect clipping for small files?)
  5. when pressing F to frame, they appear again, but again very small
  6. when pressing C to enable Auto Cull, the same behaviour when zooming / scrolling the mousewheel happens (they disappear)
 
<img width="1149" alt="image" src="https://github.com/PixarAnimationStudios/OpenUSD/assets/2693840/2f3ec76f-52ad-4c28-9a9b-2f87441ff7dd">

Expectation would be that there is
a) at least a way to configure usdview to properly display these files
b) a better default presentation of the files

### System Information (OS, Hardware)
Windows 11 and M2 MacBook Pro

**Comment by jesschimein** [comment]
Filed as internal issue #USD-8545

**Comment by drwave** [comment]
In usdview, in the menus, select Camera->Free Camera Settings…

Click “Override Near” and set it to something like 0.01

Move the camera closer. You can see it just fine.

￼

**Comment by hybridherbst** [comment]
Indeed! - but my point is that I shouldn't have to do that if there's an automatic framing at launch and an option for automatic clipping planes :)

**Comment by drwave** [comment]
Sure, I was just addressing the "at least a way to configure usdview to properly display these files” in case you didn’t realize it was a clipping plane issue - it wasn’t clear in the report.

**Comment by hybridherbst** [comment]
True, I had mistyped "clipping" as culling... fixed now!
It's a bit strange since when opening the file usdview does show something but _any_ zoom change results in nothing being visible anymore - as if the default view uses different clipping.

**Comment by spiffmon** [comment]
When you have "auto-clip" enabled, each time you do a "f" frame, it will pick a near dist that ensures you see the bounded geometry.  But zooming moves the camera without changing near-clip.  There are quite alot of heuristics involved in picking near and far in auto-clip to maximize hardware depth-buffer accuracy, but the use-case that guided those heuristics was z-fighting in really large sets, not single, very small things, and I'm not surprised it may make for some surprising behavior!

And also, related to one of your other posts, `usdrecord` does **not** have access to any of the "maximize depth buffer precision" logic, so our claim that it should be producing the same results in Storm as usdview are perhaps a bit untrue :-(

**Comment by spiffmon** [comment]
... and in many workflows, you actually would not want near to be recomputed as you zoom, because you are trying to "get past" occluding things.
